src/PanelTraits/Read.php

```diff
@@ -207,6 +207,32 @@ public function getCellView($column, $entry)
         }
     }
 
+    /**
+     * Add fields/columns names to include in select query when using ajax tables.
+     *
+     * @param string|array $fields fields/columns names.
+     */
+    public function addAjaxExtraFields($fields)
+    {
+        if (is_array($fields)) {
```

Comment:
Laravel willl probably fail if  `is_null($fields)` is true but we probably should check for it.